更新原生平台相关 (#1544)
cocos-creator/creator-docs: en/editor/publish/native-options.md

#### Template

> As of Cocos Creator 3.0, the engine template available in **Template** is **Link** for a consistent experience, removing the original **Default** template.

Reviewer: ```suggestion
Starting with Cocos Creator 3.0, the engine template available in **Template** is **Link** for a consistent experience, removing the original **Default** template.
```

> - The first time the source code engine builds and compiles a project, it takes a long time to compile C++ code, depending on the configuration of the computer, which may take 5~20 minutes. After the same project has been compiled once, the time required for the next recompile is greatly shortened.
> - The projects built by the source code engine, compiled and run using native development environment (such as Android Studio, Xcode, etc. IDE), and also can be debugged and error trapped.

> Currently, the Cocos Creator installation directory already includes Cocos2d-x source code engine in the `resources\3d\cocos2d-x-lite` folder. If you need to customize the engine, please refer to the [Engine Customization Workflow](../../advanced-topics/engine-customization.md) documentation for details.

Reviewer: ```suggestion
Currently, the Cocos Creator installation directory already includes Cocos2d-x source code engine in the `resources\3d\cocos2d-x-lite` folder. To customize the engine, please refer to the [Engine Customization Workflow](../../advanced-topics/engine-customization.md) documentation for details.
```
